packages/playwright-ct-core/src/vitePlugin.ts

```diff
@@ -48,8 +48,8 @@ export function createPlugin(): TestRunnerPlugin {
       configDir = configDirectory;
     },
 
-    begin: async (suite: Suite) => {
-      const result = await buildBundle(config, configDir, suite);
+    begin: async () => {
```

Comment:
Let's keep the unused `Suite` parameter here to mirror the `Plugin` interface.

Reply:
done